uber/utils.py

```diff
@@ -825,24 +825,27 @@ def get_unpaid_promo_code_uses_count(cls, id, already_counted_attendee_ids=None)
         return promo_code_count
 
     @classmethod
-    def to_sessionized(cls, m):
+    def to_sessionized(cls, m, name='', badges=0):
         from uber.models import Attendee, Group
         if is_listy(m):
             return [cls.to_sessionized(t) for t in m]
         elif isinstance(m, dict):
             return m
         elif isinstance(m, Attendee):
-            return m.to_dict(
+            d = m.to_dict(
                 Attendee.to_dict_default_attrs
                 + ['promo_code']
                 + list(Attendee._extra_apply_attrs_restricted))
+            d['name'] = name
+            d['badges'] = badges
+            return d
         elif isinstance(m, Group):
             return m.to_dict(
                 Group.to_dict_default_attrs
                 + ['attendees']
                 + list(Group._extra_apply_attrs_restricted))
         else:
-            raise AssertionError('{} is not an attendee or group'.format(m))
+            raise AssertionError('{} is not an attendee or dict'.format(m))
```

Comment:
Whoops, this needs to be reverted -- there was a brief, heady period of time where I had removed all the Group object stuff from the Charge class... right until I tried to load a group_members page and found we still needed it. :'D